kasa/device.py

```diff
@@ -525,19 +525,54 @@ def _get_replacing_attr(
 
         return None
 
+    def _get_deprecated_callable_attribute(self, name: str) -> Any | None:
+        vals: dict[str, tuple[ModuleName, Callable[[Any], Any], str]] = {
+            "is_dimmable": (
+                Module.Light,
+                lambda c: c.has_feature("brightness"),
+                "light_module.has_feature('brightness')",
```

Comment:
```suggestion
                'light_module.has_feature("brightness")',
```
I'd swap the quotes for all the instructions, as we are commonly preferring `"`.

Reply:
Done